CMakeLists.txt

```diff
@@ -164,47 +164,47 @@ if (XCODE)
   # Disable headermaps.
   set(CMAKE_XCODE_ATTRIBUTE_USE_HEADERMAP "NO") 
 elseif(MSVC)
-  # Disable unknown pragma warnings (e.g. for #pragma mark).
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4068")
-
-  # TODO: these warnings come from llvmSupport. Since we don't want to diverge from llvmSupport by
-  # addressing these warnings , disable these for now to clean the build log.
-  # If/when we move to use LLVM's own llvmSupport, we should reenable these warnings.
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4996") # POSIX name for this item is deprecated.
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4244") # Signed conversion.
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4267") # Possible loss of data conversions.
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4291") # Operator new with no matching delete found.
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4800") # Forcing value to bool 'true' or 'false'.
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4146") # Unary minus applied to unsigned type.
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4141") # 'inline' used more than once.
+  add_compile_options(
```

Comment:
If I understand correctly, setting CMAKE_CXX_FLAGS overrides the value of CMAKE_CXX_FLAGS_RELWITHDEBINFO.  On Windows this means that /MD doesn't get passed in. Using add_compile_options properly appends to the correct CMAKE_CXX_FLAGS variable.